InterviewFlashCardsTests/InterviewFlashCardsTests.m

```diff
@@ -22,7 +13,6 @@ @implementation InterviewFlashCardsTests
 
 - (void)setUp {
     [super setUp];
-    self.qman = [IFCQueryManager new];
```

Comment:
@mesbekmek since you've already checked this PR would you mind submitting the Review?